### PR TITLE
Feature/tweaks

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,33 @@
...
-Sites are tested every Wednesday and Friday at 00:30
+Currently the repo is setup to deploy to a Rancher environment with the
+configuration as described in [./rancher-config](./rancher-config). This will
+connect the site to the MongoDB specified by `MONGODB_URL` and set the checking
+of the websites based on the setting of `PA11Y_CRON`. A cron setting such as
+`0 30 0 * * 3,5` will run the checks at 00:30 every Wednesday and Friday.
+
+The default setting in the [Dockerfile](Dockerfile) is to start the site in
+READONLY mode. In order to add websites to the list of those to be checked,
+the more needs to be read/write. Removing the setting from the Dockerfile will
```
Comment (mcheung-nhs) [inline README.md]: the more needs to be...
should this be 'the mode needs to be...' ?

**Review by mcheung-nhs — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -2,12 +2,41 @@
...
+infrastructure is running checks on for websites you have no interest in.
+
+## CI & Deployment
+
+The application is configured for 
+[Travis CI](https://travis-ci.org/nhsuk/pa11y-dashboard) to run builds for each
+PR and build of the `master` branch. When this happens, if the initiator is a
+member of the NHS.UK GitHub organisation, a deployment will be made to the
+NHS.UK Rancher environment.
+The deployment is initiated from the [./scripts/deploy](./scripts/deploy)
```
Comment (mcheung-nhs) [inline README.md]: deploy.sh ?

File: README.md [hunk 3]
```diff
@@ -2,12 +2,41 @@
...
+connect the site to the MongoDB specified by `MONGODB_URL` and schedule the
+automated checks of the websites to run according to the setting of
+`PA11Y_CRON`.
+A cron setting such as `0 30 0 * * 3,5` will run the checks at 00:30 every
+Wednesday and Friday.
+
+The default settings in the [Dockerfile](Dockerfile) start the site in
+`READONLY` mode. In order to add websites to the list of those to be checked,
+the website needs to be in `READ/WRITE` mode. This is achieved by setting
+`READONLY=false`, as specified [above](./#running-in-development).
```
Comment (mcheung-nhs) [inline README.md]: internal link not working for 'above'

**Comment by st3v3nhunt** [comment]
Good spot. I've fixed that and made some additional wording changes.

**Review by mcheung-nhs — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (st3v3nhunt) [reply README.md]: Thanks, sorted!

File: README.md (hunk 3 above)
Comment (st3v3nhunt) [reply README.md]: Thanks, sorted!

**Review by mcheung-nhs — APPROVED** [review]

**Review by mcheung-nhs — APPROVED** [review]

**Pull request merged**